deploy/common/kata-app.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Pod
```

Comment:
@pohly Is this change related to this commit?

Reply:
No, will remove.